components/PostArticle.vue

```diff
@@ -1,62 +1,52 @@
 <template>
-	<article>
-		<h1 v-html="title" class="title" />
-		<img v-if="picture" :src="picture" class="news-item-img">
-		<WeCountLogo v-else class="news-item-img" />
-		<div v-html="content" class="api-content" />
-		<div v-for="(t, index) in tags" :key="index">
-			<nuxt-link :to="{ path: '/tag', query: { s: t }}">
-				{{ t }}
-			</nuxt-link>
-		</div>
+	<article class="post-article">
+		<h1 v-html="postToRender.title" class="title" />
+
+		<div v-html="postToRender.author" class="author" />
+
+		<div v-html="postToRender.date" class="date" />
+
+		<div v-html="postToRender.content" class="api-content" />
+
+		<section class="tags-info">
+			<p>Active Tags</p>
+			<div class="tags">
+				<nuxt-link v-for="(t, index) in postToRender.tags" :key="index" :to="{ path: '/tag', query: { s: t }}">
+					{{ t }}
```

Comment:
@lliskovoi is there a hover styling for tags? I noticed the first tag (`accessibility`) is underlined in the Figma file.

Reply:
According to [my discussion with @lliskovoi yesterday on IRC](http://irc-logs.fluidproject.org/%23fluid-design/%23fluid-design.2020-04-02.log), at 2020-04-02T16:29:22, the focus/hover styling for these tags is an underline.

Reply:
Yep, the accessibility tag shows the hover/focus styling